src/PolicyInsights/PolicyInsights/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.4.1
+* Undo bug with version 1.4.0
```

Comment:
This isn't descriptive enough.  Say something about reverting a change in 1.4.0 to pull in all pages by default.  Default page size is now 'X' for Get-AzPolicyState

Reply:
pagination is not yet supported in powershell it is to be implemented, not sure how to word this without using word page.
when a user does not provide top we return 1000 rows, once paging is supported here we will send in 1000 rows plus a nextLink.
the description for this release was support pagination, do i need to instead say "unsupport pagination"

Reply:
Let Liz/Kemley word it :-)  However, you definitely need to mention which cmdlet this affects

Reply:
Will add "Retracting changes made in powershell that increased request row limit. Removed incorrect statement of supporting paging" as suggested by Kemley